ndc-models/src/lib.rs

```diff
@@ -61,6 +61,7 @@ pub struct QueryCapabilities {
     /// Does the connector support explaining queries
     pub explain: Option<LeafCapability>,
     /// Does the connector support nested fields
+    #[serde(default)]
     pub nested_fields: NestedFieldCapabilities,
```

Comment:
Should this be `Option<NestedFieldCapabilities>`, where `None` would mean, no nested field support is available?

Reply:
No, because "no nested field support" can be indicated by not returning nested types in the schema response. Currently, if you declare nested fields, we assume you can at least select them, but we could add a sub-capability for that too.